beacon-chain/p2p/connection_gater_test.go

```diff
@@ -96,6 +98,20 @@ func TestService_InterceptBannedIP(t *testing.T) {
 	}
 }
 
+func TestService_RejectInboundConnectionBeforeStarted(t *testing.T) {
```

Comment:
Could you also add the positive case in where an inbound connection is allowed now that the service has started ?

Reply:
Good idea, I'll add it.